jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/SessionHandler.java

```diff
@@ -672,6 +680,17 @@ public void setComment(String comment)
         {
             checkAvailable();
             _sessionManager.setSessionComment(comment);
+            if (!StringUtil.isEmpty(comment))
+            {
+                if (comment.contains(SAME_SITE_STRICT_COMMENT))
+                    _sessionManager.setSameSite(HttpCookie.SameSite.STRICT);
+                if (comment.contains(SAME_SITE_LAX_COMMENT))
+                     _sessionManager.setSameSite(HttpCookie.SameSite.LAX);
+                if (comment.contains(SAME_SITE_NONE_COMMENT))
+                    _sessionManager.setSameSite(HttpCookie.SameSite.NONE);
+                if (comment.contains(PARTITIONED_COMMENT))
+                    _sessionManager.setPartitioned(true);
```

Comment:
Use the techniques in HttpCookieFacade here.
Bonus is that you can even get a clean comment field too.

Reply:
Done.